pyflakes/checker.py

```diff
@@ -1128,6 +1133,8 @@ def IMPORTFROM(self, node):
         else:
             self.futuresAllowed = False
 
+        module = ('.' * node.level) + (node.module or '')
```

Comment:
alternatively this could be

    if node.level:
        module = ('.' * node.level) + (node.module or '')
    else:
        module = node.module